exonum-dictionary.txt

```diff
@@ -59,11 +59,13 @@ nanos
 nodelay
 oneshot
 openssl
+parseable
```

Comment:
Maybe it's better to use "parsable"?

Reply:
... or not use it at all. `Unparseable` can be replaced with `invalid` imo